src/main/java/seedu/address/model/team/Team.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.team;
+
+import seedu.address.model.person.Person;
+
+public class Team {
+    private final TeamName teamName;
+    private final TeamList teamList;
+
+    Team(TeamName teamName, Person[] teamMembers) {
```

Comment:
Hi, to be consistent with teamname.java, do you want to include access modifier for the constructor and methods?